client/reader/icons/download-cloud.tsx

```diff
@@ -0,0 +1,34 @@
+const downloadCloud = (
+	<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<g clipPath="url(#clip0_3603_1982)">
+			<path
+				d="M6.66675 14.1666L10.0001 17.5L13.3334 14.1666"
+				stroke="#656970"
+				strokeWidth="2"
+				strokeLinecap="round"
+				strokeLinejoin="round"
+			/>
+			<path
+				d="M10 10V17.5"
+				stroke="#656970"
+				strokeWidth="2"
+				strokeLinecap="round"
+				strokeLinejoin="round"
+			/>
+			<path
+				d="M17.4001 15.0749C18.1246 14.5655 18.6679 13.8384 18.9511 12.9992C19.2344 12.1601 19.2429 11.2525 18.9754 10.4081C18.7079 9.5638 18.1783 8.82669 17.4635 8.30375C16.7486 7.7808 15.8858 7.49925 15.0001 7.49994H13.9501C13.6994 6.52317 13.2305 5.61598 12.5785 4.84668C11.9265 4.07737 11.1085 3.46599 10.1861 3.05857C9.26363 2.65115 8.26077 2.4583 7.25301 2.49454C6.24524 2.53078 5.25883 2.79517 4.36803 3.2678C3.47723 3.74043 2.70525 4.40898 2.11022 5.22314C1.51519 6.03729 1.11261 6.97582 0.932784 7.96807C0.752958 8.96032 0.800575 9.98044 1.07205 10.9516C1.34352 11.9228 1.83178 12.8198 2.50007 13.5749"
+				stroke="#656970"
+				strokeWidth="2"
+				strokeLinecap="round"
+				strokeLinejoin="round"
+			/>
+		</g>
+		<defs>
+			<clipPath id="clip0_3603_1982">
+				<rect width="20" height="20" fill="white" />
+			</clipPath>
+		</defs>
+	</svg>
+);
+
+export default downloadCloud;
```

Comment:
Usually SVGs in script files need to be typed with things like `import { G, Path, SVG } from '@wordpress/components';`.